src/main/java/coydir/model/person/Leave.java

```diff
@@ -0,0 +1,61 @@
+package coydir.model.person;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Leave in the database.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidLeave}
+ */
+public class Leave {
+    private static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+    public final LocalDate startDate;
+    public final LocalDate endDate;
+
+    /**
+     * Constructs a {@code Leave}.
+     *
+     * @param startDate A valid start date.
+     * @param endDate A valid start date.
+     */
+    public Leave(String startDate, String endDate) {
+        this.startDate = LocalDate.parse(startDate, FORMAT);
+        this.endDate = LocalDate.parse(endDate, FORMAT);
+    }
+
+    /**
+     * Returns true if a endDate and StartDate makes sense.
+     */
+    public static boolean isValidLeave(String startDate, String endDate) {
+        final int b = LocalDate.parse(endDate, FORMAT).compareTo(LocalDate.parse(startDate, FORMAT));
+        return b > 0;
```

Comment:
Can I just check, for `LocalDate` objects, `compareTo` makes a comparison based on the date right? Should we also accept `b = 0`, for cases like one day leaves where the start and end date are equal?

Reply:
Interesting point! I agree too